pwndbg/commands/strings.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+import argparse
+import os
+from typing import List
+import pwndbg
+import pwndbg.commands
+import pwndbg.aglib.memory
+from pwndbg.commands import CommandCategory
+
+import re
+
+from pwndbg.lib.memory import Page
+
+parser = argparse.ArgumentParser(description="Extracts and displays ASCII strings from all readable memory pages of the debugged process.")
+
+parser.add_argument(
+    "-n",                 type=int,            default=4,    help="Minimum length of ASCII strings to include")
+parser.add_argument(
+    "mapping_name",       type=str, nargs="?", default=None, help="Mapping to search [e.g. libc]")
```

Comment:
Ideally, we probably want to accept multiple mappings here, so one can execute it as `libc stack heap` or `libc libstd libm` etc.